src/gawronski-bartosz/Fsd.Bartek.Client/Fsd.Bartek.Client.Ex2/app.js

```diff
@@ -0,0 +1,8 @@
+var app = app || {};
+
+(function () { 
+    app.apiControler.initialize();
+    setTimeout(function(){
```

Comment:
królu złoty. to bieda straszna :) skąd wiesz że serwer odpowie po 750ms ? a jak odpowie po 2s ? po prostu do app.apiController.initialize przekaż callback, który wywołasz gdy dane będą gotowe.